textattack/transformations/word_swap_gradient_based.py

```diff
@@ -18,8 +19,16 @@ class WordSwapGradientBased(Transformation):
         top_n (int): the number of top words to return at each index
     """
 
-    def __init__(self, model, top_n=1):
+    def __init__(self, model_wrapper, top_n=1):
+        # Unwrap model wrappers. Need raw model for gradient.
+        if not isinstance(model_wrapper, textattack.models.wrappers.ModelWrapper):
+            raise TypeError(f"Got invalid model wrapper type {type(model_wrapper)}")
+        model = model_wrapper.model
+        self.model_wrapper = model_wrapper
+        self.model = model
```

Comment:
nit: combine this and line 26 into `self.model = model_wrapper.model`

Reply:
Also maybe set a `self.tokenizer` so you only have to change it in one place if you change it again?

Reply:
good idea